src/Assistant.ts

```diff
@@ -46,6 +46,7 @@ type SetSuggestedPromptsFn = (
 
 interface SetSuggestedPromptsArguments {
   prompts: [AssistantPrompt, ...AssistantPrompt[]];
+  title?: string;
```

Comment:
@filmaj It appears that this [already exists](https://github.com/slackapi/node-slack-sdk/blob/main/packages/web-api/src/types/request/assistant.ts#L22) in the Node SDK, so nothing should be required there by way of changes. Let me know if that's a bad assumption!

Reply:
Will incorporate!